modules/server/templates/scripts/forseti-server/run_forseti.sh.tpl

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /home/ubuntu/forseti_env.sh
+
+
+# set -x enables a mode of the shell where all executed commands are printed to the terminal.
+# With this  enabled, we should not put anything private/secret in the commands called because
+# they will be logged.
+set -x
+
+# Remove any CAI temp files generated by the forseti process.
+# Failed inventory due to various reasons could cause the temp files not being deleted.
+sudo rm -rf /tmp/forseti-cai-*
+
+# Put the config files in place.
+sudo gsutil cp gs://$SCANNER_BUCKET/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
```

Comment:
I would recommend that for each `$SCANNER_BUCKET` you pass in a Terraform variable.  We could conceivably remove the need of the `forseti_env.sh` file.

Reply:
For example, you can change line 29 as follows:
```
sudo gsutil cp gs://$SCANNER_BUCKET/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
```
```
sudo gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
```
Then in
https://github.com/forseti-security/terraform-google-forseti/blob/eceeacf0e24b437aad0b2f039bf328ec0945f1e3/modules/server/main.tf#L123-L132
You can add a variable
```
storage_bucket_name              = var.server_gcs_module.forseti-server-storage-bucket

Reply:
Updated to use the TF var for `SCANNER_BUCKET` and removed `forseti_env.sh` script.